tsconfig.json

```diff
@@ -19,7 +19,7 @@
     "allowJs": true,
     "skipLibCheck": true,
     "strict": false,
-    "noEmit": true,
+    "noEmit": false,
```

Comment:
"emitting" means that the build TS files are actually placed in the folder specified above (`outDir`)